docs/content/en/docs/implementing/tasks/_index.md

```diff
@@ -5,12 +5,72 @@ weight: 90
 hidechildren: false # this flag hides all sub-pages in the sidebar-multicard.html
 ---
 
-Keptn tasks are defined in a
+A
 [KeptnTaskDefinition](../../yaml-crd-ref/taskdefinition.md/)
-resource.
-A task definition includes a function
+resource defines tasks that the Keptn Lifecycle Toolkit runs
+as part of the pre- and post-deployment phases of a
+[KeptnApp](../../yaml-crd-ref/app.md) or
+[KeptnWorkload](../../concepts/workloads.md).
+
+A Keptn task executes as a
+[runner](https://docs.gitlab.com/runner/executors/kubernetes.html#how-the-runner-creates-kubernetes-pods)
```

Comment:
I would not link gitlab documentation here. Gitlab has a pipeline system that also has a similar concept of having runners to perform CI tasks. However, this documentation is for setting their CI runner up 
```suggestion
runner
```

Reply:
Done